Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -965,6 +965,7 @@ using McParticle = McParticles::iterator;
 namespace soa
 {
 DECLARE_EQUIVALENT_FOR_INDEX(aod::StoredMcParticles_000, aod::StoredMcParticles_001);
+DECLARE_EQUIVALENT_FOR_INDEX(aod::StoredTracks, aod::StoredTracksIU);
```

Comment:
Shouldn't we do the same for the Cov matrix?

Reply:
I asked the same question to @aalkin but apparently the counter is based solely on StoredTracks (and that's what is used for the cov matrices) so that's not necessary! @aalkin , could you confirm, to be sure? Thanks!

Reply:
Yes, the base for the index is the same as the base for the join, it is enough to only declare equivalence for the tracks. TracksCovs and others are never (expected to be) used not in a join with Tracks.

Reply:
I see. Thanks